modules/core/src/main/scala/com.snowplowanalytics.snowplow.snowflake/Metrics.scala

```diff
@@ -77,5 +114,16 @@ object Metrics {
       val metricType = CommonMetrics.MetricType.Gauge
     }
 
+    final case class LatencyCollectorToTargetMillis(v: Long) extends CommonMetrics.KVMetric {
+      val key        = "latency_collector_to_target_millis"
+      val value      = v.toString
```

Comment:
Not about conformity to the spec but it's counterintuitive that this is cast to string - surely it's an int?

Reply:
reporters use this `value` directly in string concatenation when building a metric to statsd or stdout, you can see that `latency_millis` is also casted to String